lib/food_truck.rb

```diff
@@ -0,0 +1,22 @@
+class FoodTruck
+  attr_reader :name, :inventory
+
+  def initialize(name)
+    @name = name
+    @inventory = Hash.new(0)
+  end
+
+  def check_stock(item)
+    @inventory[item]
+  end
+
+  def stock(item, quantity)
+    @inventory[item] += quantity
+  end
+
+  def potential_revenue
+    @inventory.sum do |item, quantity|
+      item.price * quantity
+    end
+  end
+end
```

Comment:
This whole class looks awesome! Nice work with the Hash default value and the .sum enumerable.